src/main/java/domain/Racing.java

```diff
@@ -0,0 +1,19 @@
+package domain;
+
+public class Racing {
+
+    private final Cars carList;
```

Comment:
사소하지만, 변수이름에 Collection 구현체 담지 마세요! 
나중에 Set으로 수정하면 이름도 같이 바꿔야 해서요...!
```suggestion
    private final Cars cars;
```